shared/util/errors.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+class RPCError extends Error {
```

Comment:
FYI this won't work as you'd expect: http://stackoverflow.com/questions/30402287/extended-errors-do-not-have-message-or-stack-trace

Just ran into this on another branch. Looks like https://www.npmjs.com/package/babel-plugin-transform-builtin-extend is a good fix for this.